README.md

```diff
@@ -100,10 +100,39 @@ To run you will use the "shares" command
 
 ```bash
 # single file
-yarn cli shares --keystore=keystore.json --password=test --operator-ids=1,2,3,4 --operator-keys=LS..,LS..,LS..,LS.. --output-folder=./ --owner-address=... --owner-nonce=..
+yarn cli shares --keystore=./keystore.json --password=./password.txt --operator-ids=1,2,3,4 --operator-keys=LS..,LS..,LS..,LS.. --output-folder=./ --owner-address=... --owner-nonce=..
 
 # folder with multiple keystore files
-yarn cli shares --keystore=./keystore-files --password=test --operator-ids=1,2,3,4 --operator-keys=LS..,LS..,LS..,LS.. --output-folder=./ --owner-address=... --owner-nonce=..
+## folder structure for keystore files with the same password
+keystore-files
+├── keystore1.json
+├── keystore2.json
+├── keystore3.json
+├── keystore4.json
+└── keystore5.json
+password.txt
+
+yarn cli shares --keystore=./keystore_files --password=./password.txt --operator-ids=1,2,3,4 --operator-keys=LS..,LS..,LS..,LS.. --output-folder=./ --owner-address=... --owner-nonce=..
+
+## folder structure for keystore files with diffent passwords
+keystore_folders
+├── validator1
+│   ├── keystore.json
+│   └── password.txt
+├── validator2
+│   ├── keystore.json
+│   └── password.txt
+├── validator3
+│   ├── keystore.json
+│   └── password.txt
+├── validator4
+│   ├── keystore.json
+│   └── password.txt
+└── validator5
+    ├── keystore.json
+    └── password.txt
+
+yarn cli shares --keystore=./keystore_folders --password=password.txt --operator-ids=1,2,3,4 --operator-keys=LS..,LS..,LS..,LS.. --output-folder=./ --owner-address=... --owner-nonce=..
```

Comment:
instead "password.txt" write <password file name> here and in the folder structure - the name and extension are not important as long as file can be read as utf-8 encoded text